lib/aggregate.js

```diff
@@ -579,6 +771,63 @@ function sendResponseCounter(adapter, msg, options, data, startTime) {
     }
 }
 
+/**
+ * Get percentile value from an array.
+ *
+ * @param {Number} p - percentile
```

Comment:
do we really want to work with values like 50 percentile? I would prefer range 0 - 1. If out of range, throw/log (whatever is history fashion)